### DISCUSSION TITLE
Project Governance Model

### DISCUSSION THREAD

**Discussion by jacques-n** [substrait-io/substrait#16] [General]
@emkornfield opened issue #13 asking whether it makes to clarify a longer term governance structure for the project. I think it's a good question so opening up a discussion here about it. My goals for substrait:

* Substrait should be a consensus-driven project where all collaboration and decision-making is done in the public.
* Substrait should not be overly controlled/influenced by any particular organization/corporation.
* Users of substrait should be confident that the project won't one day "change its stripes" like Redis, Confluent or various other projects did.

I think there are several different foundations under which substrait can operate to accomplish these. ASF is one of them. Really want to see the community form and make a group decision about which model/requirements makes sense rather prescribe a particular approach. Given that, would love people to provide opinions on this discussion thread of key governance aspects that they would like to see and/or are important to them.

**Comment by emkornfield** [comment]
Thanks for starting the discussion here, I think the bullet points you list are my primary concerns (and will hopefully help with adoption if we can achieve them).

**Comment by rdblue** [comment]
I agree with all 3 points. The foundation is less important than than neutrality, consensus, and reliability.

**Reply by curino** [reply]
@rdblue, I completely agree that the substance of neutrality/consensus/reliability is what truly matters, but a good choice of foundation is one (of few good) way(s) to signal to folks that are not yet involved in the project what the ethos of the project is all about. Kind of like a short-hand for all the things we love of healthy OSS communities. 

This is key for adoption because it takes a lot of contact-surface with an OSS community (discussions, PRs, etc.) to gauge how true those attributes are in a project, and I suspect lots of instituitions/projects that could benefit from using/contributing to substrait might not have the foresight or ability to invest the needed effort to gauge directly. Strong signaling (bylaws/foundations/explicit rules for commitership/etc.) could reassure folks their time/effort investments are likely to pay-out.
